examples/simple/config/base.yaml

```diff
@@ -1,6 +1,6 @@
-applicationID: simple
-applicationDesc: A simple example service
-applicationOwner: owner@service.com
+name: simple
+description: A simple example service
+owner: owner@service.com
```

Comment:
Thought about all the options and decided to strip away service and application for 2 reasons:
1. Making them top level constructs, it indicates they are metadata about the repo. At the top level, a name, owner and description indicate it is about the service.
2. To avoid confusion with users considering it a services vs application vs something else. At times, there are multiple services (thrift services for eg.) within a repo.